README.md

```diff
@@ -86,6 +86,11 @@ See [getting started](https://meshery.io/#getting-started) to quickly deploy Mes
           <td><a href="https://github.com/layer5io/meshery-cpx">
             <img src='https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQksHj15DkID308qQw3cmkQrRULPxyzbVquSZVev-9dj1L6sPs-rQ&s' alt='Citrix CPX Service Mesh' align="middle" hspace="5px" vspace="5px" height="30px" width="30px">Meshery adapter for Citrix CPX</a>
           </td>
```

Comment:
Thanks for this. One caveat relocating rows is the need to update the `rowspan` count in that section. We will want to update it to "3" in line 85 and to "5" on line 97.

Reply:
My bad , thanks God for reviews. 😆